<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Image</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-new">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="formatComboBox">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="sizeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Si&amp;ze</string>
     </property>
     <property name="buddy">
      <cstring>widthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="paletteComboBox">
     <item>
      <property name="text">
       <string>Arne 16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DawnBringer 16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DawnBringer 32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Commodore 64 (ptoing)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Commodore 64 (Pepto)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EGA Standard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EGA Global</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NES</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&amp;Open...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="formatLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>For&amp;mat</string>
     </property>
     <property name="buddy">
      <cstring>formatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="paletteLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Pa&amp;lette</string>
     </property>
     <property name="buddy">
      <cstring>paletteComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="heightSpinBox">
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="singleStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="presetButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Preset</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addPresetButton">
       <property name="text">
        <string>Add Preset</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff/>
        </iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removePresetButton">
       <property name="text">
        <string>Remove Preset</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff/>
        </iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>paletteComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
